classes/range.js

```diff
@@ -81,8 +81,8 @@ class Range {
 
     // memoize range parsing for performance.
     // this is a very hot path, and fully deterministic.
-    const memoOpts = Object.keys(this.options).join(',')
-    const memoKey = `parseRange:${memoOpts}:${range}`
+    const memoOpts = buildMemoKeyFromOptions(this.options)
```

Comment:
```suggestion
    const memoOpts =
      (this.options.includePrerelease && FLAG_INCLUDE_PRERELEASE) |
      (this.options.loose && FLAG_LOOSE) |
      (this.options.rtl && FLAG_RTL)

```

Reply:
Should we keep rtl flag even if is not used?

Reply:
javascript is fun!

```javascript
> undefined | undefined | undefined
0
> false | false | false
0
```

Reply:
> Should we keep rtl flag even if is not used?

Sorry no, that was me working off your PR before you removed it. Updated my suggestion.